index.js

```diff
@@ -10,7 +10,7 @@ const runShellCmd = async (command, failureMessage) => {
         errorMessage += `\nStandard Error: ${stderr}`;
       if (error || errorMessage)
         core.setFailed(errorMessage);
-      core.debug(stdout);
+      console.log(stdout);
```

Comment:
Is leaving this logging the detailed info on every run intended here?  If not, we should switch it back to core.debug

Reply:
I think it saves time to see the info the first time, rather than having to re-run in debug mode. There isn't excessive output to wade through.